internal/k8s/converter.go

```diff
@@ -94,7 +95,9 @@ type UnstructuredConverter struct {
 }
 
 // NewUnstructuredConverter returns a new UnstructuredConverter initialized
-func NewUnstructuredConverter() *UnstructuredConverter {
+// TODO(youngnick): Remove the boolean once the service-apis code is not
+// experimental.
+func NewUnstructuredConverter(addServiceApis bool) (*UnstructuredConverter, error) {
```

Comment:
is the check necessary? Why not always convert service api types? This should be safe to do, and one less dimension we need to test.

Reply:
After thinking about it for a bit, I agree. I'll remove. The only way that code will ever be hit is if we register the informers in `serve.go` anyway.